ontology/coap.ttl

```diff
@@ -0,0 +1,164 @@
+                                                      schema:url "https://github.com/JKRhb/"
+                                                    ] ;
+                                                 schema:name "Jan Romann"
+                                               ] ;
+                                    vann:preferredNamespacePrefix "cov" ;
+                                    vann:preferredNamespaceUri : ;
+                                    rdfs:comment "Vocabulary to represent CoAP messages in RDF."@en ;
+                                    rdfs:label "CoAP in RDF"@en .
+
+_:genid4 rdf:type schema:Organization ;
+          schema:name "Siemens AG" ;
+          schema:url "https://www.siemens.com/" .
+
+#################################################################
+#    Object Properties
+#################################################################
+
+
+:hasMethod rdf:type owl:ObjectProperty ;
+             rdfs:domain :Request ;
+             rdfs:comment "Indicates the CoAP method to be used in the request."@en ;
+             rdfs:isDefinedBy <> ;
+             rdfs:seeAlso "See the <a href=\"https://www.iana.org/assignments/core-parameters/#method-codes\">CoAP Method Codes</a> registry for the set of possible values.";
+             rdfs:label "hasMethod"@en .
+
+:hasBlockwise rdf:type owl:ObjectProperty ;
+        rdfs:domain :Message ;
+        rdfs:comment "Indicates that block-wise transfer [[RFC7959]] is supported."@en ;
+        rdfs:isDefinedBy <> ;
+        rdfs:label "hasBlockwise"@en .
+
+:hasQBlockwise rdf:type owl:ObjectProperty ;
+        rdfs:domain :Message ;
+        rdfs:comment "Indicates that quick block-wise transfer [[RFC9177]] is supported."@en ;
+        rdfs:isDefinedBy <> ;
+        rdfs:label "hasQBlockwise"@en .
+
+:hasHopLimit rdf:type owl:ObjectProperty ;
+        rdfs:domain :Request ;
+        rdfs:comment "Indicates that a CoAP Hop-Limit option [[RFC8768]] should be included in the request."@en ;
+        rdfs:isDefinedBy <> ;
+        rdfs:label "hasHopLimit"@en .
+
+:hasContentFormat rdf:type owl:ObjectProperty ;
+        rdfs:domain :Message ;
+        rdfs:comment "Indicates a numeric CoAP Content-Format [[RFC7252]] as a combination of Media Type and Content Coding. Must match the given <code>contentType</code> and <code>contentCoding</code> members."@en ;
+        rdfs:isDefinedBy <> ;
+        rdfs:label "hasContentFormat"@en .
+
+:hasAccept rdf:type owl:ObjectProperty ;
+        rdfs:domain :Request ;
+        rdfs:comment "Indicates that a Consumer must include an Accept option [[RFC7252]] with the given value when requesting the resource associated with this Form."@en ;
+        rdfs:isDefinedBy <> ;
+        rdfs:label "hasAccept"@en .
+
+#################################################################
+#    Data properties
+#################################################################
+
+:BlockWiseTransferParameters rdf:type owl:DatatypeProperty ;
```

Comment:
Is `:BlockWiseTransferParameters` intended to be a datatype instead of a datatype property?

```suggestion
:BlockWiseTransferParameters rdf:type rdfs:Datatype ;
```

What's the value space of `:BlockWiseTransferParameters` literals? Wouldn't a class make more sense?

Reply:
Good point, updated in https://github.com/w3c/wot-binding-templates/commit/4e29228500359ab12907dbe0e70c248a5a850742

Reply:
`:BlockWiseTransferParameters` in coap.ttl now looks like this:

https://github.com/w3c/wot-binding-templates/blob/e71a3954aecd94bd4f45ca0eae3c263b7c1282fa/ontology/coap.ttl#L122-L126

It's defined both as an instance of `owl:Class` (line 122) and to have a domain (line 123), which makes it an instance of `rdfs:Property`. Should line 123 be removed´?

Also, should `:hasBlockwise` and `:hasQBlockwise` have `:BlockWiseTransferParameters` as range?